tensorflow/contrib/metrics/python/ops/metric_ops.py

```diff
@@ -3326,8 +3329,10 @@ def cohen_kappa(labels, predictions, num_classes, weights=None,
   doesn't support weighted matrix yet.
 
   Args:
-    labels: 1-D `Tensor` of the ground truth values.
-    predictions: 1-D `Tensor` of the predicted values.
+    labels: 1-D `Tensor` of real labels for the classification task. Must be
+      one of the following types: int16, int32, int64, float32, float64.
```

Comment:
Can we drop float? I thought these are class indices. Also, for integer predictions, we prefer the arg name `predictions_idx`, as in precition_at_top_k.